recipe/migrations/example.exyaml

```diff
@@ -1,9 +1,16 @@
-migrator_ts: 0  # The timestamp of when the migration was made
-__migrator: 
-  kind: 
+# The timestamp of when the migration was made
+# Can be obtained by copying the output of
+# python -c "import time; print(f'{time.time():.0f}')"
+migrator_ts: 0
+__migrator:
+  kind:
     version
-  migration_number:  # Only use this if the bot messes up, putting this in will cause a complete rerun of the migration
-    1 
+  # Only use the migration_number if the bot messes up,
+  # putting this in will cause a complete rerun of the migration
+  migration_number:
+    1
+
+# The name of the feedstock you wish to migrate
 python:
-  - 3.6
-  - 3.8a1
+  - 3.6    # old version
```

Comment:
This is not necessarily true, in this case we're building 3.6 and 3.8a1. The value here is whatever you want the resulting config to be.